pkgs/development/python-modules/kinparse/default.nix

```
@@ -0,0 +1,35 @@
+{ stdenv
+, python3Packages
+, fetchFromGitHub
+}:
+
+python3Packages.buildPythonPackage {
```

when adding packages to `python-modules`, the interpreter is parameterized for you, so you should only bring in python packages, and not an interpreter-specific version. This will also cause issues if you want to do somethin like `python37Packages.kinparse`, and the all the dependencies are from python38
```suggestion
buildPythonPackage {
```